### PR TITLE
Prepare release 8.0.0

### PR DESCRIPTION
**Pull request by boblangley**

### CODE REVIEW

File: src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj [hunk 1]
```diff
@@ -1,29 +1,33 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <DebugType>full</DebugType>
-    <DebugSymbols>True</DebugSymbols>
+    <TargetFrameworks>net452</TargetFrameworks>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```
Comment (SimonCropp) [inline src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj]: isnt this only for projects that tests netcore?

File: src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
+    <PackageReference Include="NUnit" Version="3.7.1" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
+    <PackageReference Include="ApiApprover" Version="6.1.0-beta2" />
+    <PackageReference Include="PublicApiGenerator" Version="6.1.0-beta2" />
+    <PackageReference Include="ApprovalTests" Version="3.*" />
+    <PackageReference Include="ApprovalUtilities" Version="3.*" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj]: we already have one of these below?

File: src/NServiceBus.Spring/NServiceBus.Spring.csproj [hunk 3]
```diff
@@ -3,16 +3,12 @@
   <PropertyGroup>
     <TargetFrameworks>net452</TargetFrameworks>
     <AssemblyName>NServiceBus.ObjectBuilder.Spring</AssemblyName>
-    <NuSpecFileName>NServiceBus.Spring</NuSpecFileName>
+    <RootNamespace>NServiceBus.ObjectBuilder.Spring</RootNamespace>
```
Comment (SimonCropp) [inline src/NServiceBus.Spring/NServiceBus.Spring.csproj]: i would just change the project name

File: src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj [hunk 4]
```diff
@@ -1,29 +1,33 @@
...
-    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
-    <PackageReference Include="NUnit" Version="3.*" />
-    <PackageReference Include="ApiApprover" Version="6.1.0-*" />
-    <PackageReference Include="PublicApiGenerator" Version="6.1.0-*" />
-    <PackageReference Include="NServiceBus.ContainerTests.Sources" Version="[7.0.0-*,8.0)" />
-    <ProjectReference Include="..\NServiceBus.Spring\NServiceBus.Spring.csproj" />
+    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
+    <PackageReference Include="NUnit" Version="3.7.1" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
```
Comment (SimonCropp) [inline src/NServiceBus.Spring.Tests/NServiceBus.Spring.Tests.csproj]: no netcore so the extra group is redundant

File: src/NServiceBus.Spring.AcceptanceTests/NServiceBus.Spring.AcceptanceTests.csproj [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\NServiceBus.Spring\NServiceBus.Spring.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="7.0.0-*" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="7.0.0-*" />
     <PackageReference Include="Common.Logging" Version="3.*" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring.AcceptanceTests/NServiceBus.Spring.AcceptanceTests.csproj]: i think u can drop all Common.Logging refs and let that be inferred by nuget.

File: src/NServiceBus.Spring/NServiceBus.Spring.csproj [hunk 6]
```diff
@@ -21,10 +17,13 @@
...
   <ItemGroup>
     <PackageReference Include="Spring.Core" Version="[2.0.1, 3.0.0)" />
-    <PackageReference Include="Janitor.Fody" Version="1.*" PrivateAssets="all" />
     <PackageReference Include="NServiceBus" Version="[7.0.0-*, 8.0.0)" />
-    <PackageReference Include="Obsolete.Fody" Version="4.*" PrivateAssets="all" />
-    <PackageReference Include="Fody" Version="2.*" PrivateAssets="all" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="all" />
+    <PackageReference Include="Fody" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Janitor.Fody" Version="1.5.1" PrivateAssets="All" />
+    <PackageReference Include="Obsolete.Fody" Version="4.2.4" PrivateAssets="All" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring/NServiceBus.Spring.csproj]: u can drop this ref

File: src/NServiceBus.Spring/NServiceBus.Spring.csproj [hunk 7]
```diff
@@ -21,10 +17,13 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Spring.Core" Version="[2.0.1, 3.0.0)" />
-    <PackageReference Include="Janitor.Fody" Version="1.*" PrivateAssets="all" />
     <PackageReference Include="NServiceBus" Version="[7.0.0-*, 8.0.0)" />
-    <PackageReference Include="Obsolete.Fody" Version="4.*" PrivateAssets="all" />
-    <PackageReference Include="Fody" Version="2.*" PrivateAssets="all" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="all" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring/NServiceBus.Spring.csproj]: should be a wildcard

File: src/NServiceBus.Spring/NServiceBus.Spring.csproj [hunk 8]
```diff
@@ -21,10 +17,13 @@
...
+    <PackageReference Include="Fody" Version="2.1.2" PrivateAssets="All" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring/NServiceBus.Spring.csproj]: this and below should be a minor wildcard range

File: src/NServiceBus.Spring.AcceptanceTests/NServiceBus.Spring.AcceptanceTests.csproj [hunk 9]
```diff
@@ -1,20 +1,20 @@
...
     <PackageReference Include="Common.Logging.Core" Version="3.*" />
     <PackageReference Include="Spring.Core" Version="2.*" />
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="[7.0.0-*,8.0)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8.0)" />
-    <PackageReference Include="NUnit" Version="3.*" />
-    <ProjectReference Include="..\NServiceBus.Spring\NServiceBus.Spring.csproj" />
+    <PackageReference Include="NUnit" Version="3.7.1" />
```
Comment (SimonCropp) [inline src/NServiceBus.Spring.AcceptanceTests/NServiceBus.Spring.AcceptanceTests.csproj]: prob a minor wild card would suffice here

File: src/NServiceBus.Spring/NServiceBus.Spring.csproj (hunk 3 above)
Comment (boblangley) [reply src/NServiceBus.Spring/NServiceBus.Spring.csproj]: I prefer to keep this here, or update usings, over changing the project name. @Particular/container-maintainers do you have a preference?

Comment (WilliamBZA) [reply src/NServiceBus.Spring/NServiceBus.Spring.csproj]: No preference from my side.

Comment (SimonCropp) [reply src/NServiceBus.Spring/NServiceBus.Spring.csproj]: sorry. this should no derail the PR.

**Pull request merged**